src/mlpack/methods/ann/layer/elu_impl.hpp

```diff
@@ -49,15 +52,18 @@ void ELU<InputDataType, OutputDataType>::Forward(
     const InputType&& input, OutputType&& output)
 {
   Fn(input, output);
+
+  if (not deterministic)
```

Comment:
I didn't know that `not` was even a valid C++ keyword. :)  Since we use `!` everywhere else in the code, do you want to make this consistent with that and use `if (!deterministic)`?

Reply:
Yeah sure. Even I came across this recently. `C++` even has support for `or`, `and`, `xor` and so on. It's quite useful in technical interviews, makes the actual code look like pseudocode.